cluster-provision/gocli/cmd/run.go

```diff
@@ -956,14 +973,17 @@ func provisionNode(sshClient libssh.Client, n *nodesconfig.NodeLinuxConfig) erro
 		opts = append(opts, realtimeOpt)
 	}
 
-	for _, s := range soundcardPCIIDs {
-		// move the VM sound cards to a vfio-pci driver to prepare for assignment
-		bvfio := bindvfio.NewBindVfioOpt(sshClient, s)
-		opts = append(opts, bvfio)
+	// sound cards are not supported on s390x.
+    if runtime.GOARCH != "s390x" {
```

Comment:
please use tabs for indentation

Reply:
Thanks. Actually I've couple of more places, where I fixed this indentation issue after you pointed this. I will take of such issues in future.

Reply:
Not sure why spaces slippped in here - I remember that gofmt emits tabs as default

Reply:
right. Not sure, but today I've (re-)installed all gotools and now I see it is working.